3rdParty/CMakeLists.txt

```diff
@@ -2,9 +2,21 @@ add_subdirectory(EndianPortable)
 add_subdirectory(Getopt-for-Visual-Studio)
 add_subdirectory(hash-library)
 add_subdirectory(json)
-add_subdirectory(LightPcapNg)
 add_subdirectory(MemPlumber/MemPlumber)
 
+include(FetchContent)
+
+set(LIGHT_PCAPNG_BUILD_TYPE OBJECT CACHE STRING "" FORCE)
+
+FetchContent_Declare(
+  LightPcapNg
+  GIT_REPOSITORY https://github.com/PcapPlusPlus/LightPcapNg
+  GIT_TAG
+    copy-from-pcapplusplus # TODO: Replace with master
```

Comment:
I'll replace it with `main` after we merge this PR: https://github.com/PcapPlusPlus/LightPcapNg/pull/1

Reply:
Updated here: https://github.com/seladb/PcapPlusPlus/pull/1792/commits/bb3bd1df82a6c0ed99c4608fbb510e27fb8902d8